packages/engine-core/src/binary/BinaryEngine.ts

```diff
@@ -602,11 +602,11 @@ ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
               json.fields?.message?.startsWith('Started http server')
             ) {
               if (this.useUds) {
-                this.connection.open('http://localhost', {
+                this.connection.open('http://127.0.0.1', {
```

Comment:
FWIW, I don't think it means anything in the context of Unix Domain Sockets, but I changed it as well for consistency.